packages/ml/mobilenet/training/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM wpilib/axon-base:latest
+
+RUN mkdir -p /tensorflow/models/research/learn && \
+    mkdir -p /tensorflow/models/research/learn/ckpt && \
+    rm -rf /tensorflow/models/research/learn/train && \
+    rm -rf /tensorflow/models/research/learn/models && \
+    mkdir -p /tensorflow/models/research/start_ckpt  && \
+    cd /tensorflow/models/research/start_ckpt && \
+    wget "https://github.com/wpilibsuite/DetectCoral/releases/download/v2/model.ckpt.data-00000-of-00001" && \
```

Comment:
Not sure where to host this.

Reply:
I guess we could do it here right?

Reply:
Hosting in a release is quirky. Maybe wpilib has cloud storage somewhere